dev/bots/test/prepare_package_test.dart

```diff
@@ -342,6 +344,96 @@ void main() {
         const JsonEncoder encoder = JsonEncoder.withIndent('  ');
         expect(contents, equals(encoder.convert(jsonData)));
       });
+
+      test('publishArchive throws if forceUpload is false and artifact '
+           'already exists on cloud storage', () async {
```

Comment:
nit: We usually don't wrap test description lines. (makes them easier to grep for)

Reply:
makes sense

Reply:
done